src/gc.c

```diff
@@ -259,6 +259,20 @@ static int check_timeout = 0;
 static gcpage_t *page_metadata(void *data);
 static void pre_mark(void);
 static void post_mark(arraylist_t *list, int dryrun);
+static region_t *find_region(void *ptr, int maybe);
+jl_taggedvalue_t *jl_gc_find_taggedvalue_pool(char *p, size_t *osize_p);
```

Comment:
@tkelman Should be fixed by https://github.com/JuliaLang/julia/commit/24f66470f4d12cf47d45f254d81592b88798e434 . I'll push to master after the branch travis passes.